adoc/deployment-bare-metal.adoc

```diff
@@ -28,39 +28,75 @@ Refer to the official {ay} documentation for more information: link:https://www.
+.. `<username>sles</username>`
 +
-Refer to the official {ay} documentation for more information: link:https://www.suse.com/documentation/sles-15/singlehtml/book_autoyast/book_autoyast.html[{ay} Guide].
-. Update the file with your desired username and ssh keys. Below we disable `root` and add a `sles` user with no password.
+Insert your authorized key in the placeholder field.
+.. `<users>`
 +
-----
-  <users config:type="list">
-    <user>
-      <username>root</username>
-      <user_password>!</user_password>
-      <encrypted config:type="boolean">true</encrypted>
-    </user>
-    <user>
-      <username>sles</username>
-      <user_password>!</user_password>
-      <encrypted config:type="boolean">true</encrypted>
-      <authorized_keys config:type="list">
-        <authorized_key>[ssh public key]</authorized_key>
-      </authorized_keys>
-    </user>
-  </users>
-----
+You can add additional users by creating new blocks in the configuration containing their data.
 +
 [NOTE]
 ====
-If your users don't have a password like the above, ensure your user's `sudoers` file is updated so that you can use sudo without a password.
-The default {ay} file provides an example for this scenario.
+If the users are configured to not have a password like in the example, ensure the system's `sudoers` file is updated.
+Without updating the sudoers file the user will only be able to perform basic operations that will prohibit many administrative tasks.
+
+The default {ay} file provides examples for a disabled `root` user and a `sles` user with authorized key SSH access.
 
 The password for root can be enabled by using the `passwd` command.
 ====
-. Host the {ay} file on a webserver reachable inside the network your are installing the cluster in.
-. Provide `autoyast=https://[webserver/path/to/autoyast.xml]` during the {sle} 15 SP1 installation.
+.. `<suse_register>`
++
+Insert the email address and {productname} registration code in the placeholder fields. This activates {sle} 15 SP1.
+.. `<addon>`
++
+Insert the {productname} registration code in the placeholder field. This enables the {productname} extension module.
+Update the {ay} file with your registration keys and company best practices and hardware configurations.
++
+[NOTE]
+====
+Your {productname} registration key can be used to both activate {sle} 15 SP1 and enable the extension.
+====
+
++
+Refer to the official {ay} documentation for more information: link:https://www.suse.com/documentation/sles-15/singlehtml/book_autoyast/book_autoyast.html[{ay} Guide].
++
+. Host the {ay} files on a webserver reachable inside the network your are installing the cluster in.
+
+=== Provisioning The Cluster Nodes
+
+Once the {ay} file is available in the network of the cluster you can start deploying machines.
+
+The default production scenario consists of 8 nodes:
+
+* 2 Load Balancers
```

Comment:
I don't think that `skuba` command supports more than one load-balancer.

Reply:
This is how it's described in the architecture description. Any idea how this would be handled?

Reply:
No idea, maybe it can be achievable by using round-robin DNS entry (two lb hosts with the same public IP) but I would describe only a case with 1 LB. After clarifying we can modify again.

Reply:
I talked to Rafa we need to clarify the load balancer stuff more.